source/application/CLS/CLSAppController.h

```diff
@@ -21,10 +21,55 @@ class CLSAppController : public AMAppController
 		BioXASSideBeamlineId = 7,
 		BioXASMainBeamlineId = 8,
 		BioXASImagingBeamlineId = 9,
-		SXRMBBeamlineId = 10
+		SXRMBBeamlineId = 10,
 		// new beamlines need to be added here ...
+		CLSMAXBeamlineId
 	};
 
+	/// returns the beamline name of a given beamline Id
+	static QString beamlineName(CLSAppController::CLSBeamlineID beamline)
+	{
+		QString name;
+		switch (beamline) {
+		case BlankFacility:
+			name = "";
+			break;
+		case PreserverdALS801Beamline:
```

Comment:
@helfrij again, maybe @dretrex can answer this question. I named the beamline ID 1 and 2 as  `BlankFacility` and `PreserverdALS801Beamline`, which might not be accurate. As mentioned above, those two beamline IDs are general Ids, which I'm not aware of the usage.